lib/plugins/chat.js

```diff
@@ -191,11 +219,61 @@ function inject (bot, options) {
   bot.tabComplete = tabComplete
 
   function addDefaultPatterns () {
-    if (!defaultChatPatterns) return
-    const USERNAME_REGEX = '(?:\\(.{1,15}\\)|\\[.{1,15}\\]|.){0,5}?(\\w+)'
+    // 1.19 changes the chat format to move <sender> prefix from message contents to a seperate field.
+    // TODO: new chat lister to handle this
     bot.addChatPattern('whisper', new RegExp(`^${USERNAME_REGEX} whispers(?: to you)?:? (.*)$`), { deprecated: true })
     bot.addChatPattern('whisper', new RegExp(`^\\[${USERNAME_REGEX} -> \\w+\\s?\\] (.*)$`), { deprecated: true })
-    bot.addChatPattern('chat', new RegExp(`^${USERNAME_REGEX}\\s?[>:\\-»\\]\\)~]+\\s(.*)$`), { deprecated: true })
+    bot.addChatPattern('chat', LEGACY_VANILLA_CHAT_REGEX, { deprecated: true })
+  }
+
+  bot.awaitMessage2 = (opts) => {
```

Comment:
hmm why awaitMessage and awaitMessage2 ?

Reply:
awaitMessage2 has more options and can match message content/sender without using regex on 1.19+